pkgs/servers/monitoring/telegraf/gomod.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/go.mod b/go.mod
+index 1f6bd6b8..c2090ddb 100644
+--- a/go.mod
++++ b/go.mod
+@@ -110,7 +110,7 @@ require (
+ 	github.com/safchain/ethtool v0.0.0-20200218184317-f459e2d13664
+ 	github.com/samuel/go-zookeeper v0.0.0-20180130194729-c4fab1ac1bec // indirect
+ 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b // indirect
+-	github.com/shirou/gopsutil v2.20.5+incompatible
++	github.com/shirou/gopsutil v2.20.6+incompatible
```

Comment:
Did they solve the vendoring problem in this release? I can make a telegraf PR for this as well.

Reply:
Yes, the vendoring was fixed in https://github.com/shirou/gopsutil/pull/889.